libfreerdp/core/gateway/http.c

```diff
@@ -1128,7 +1127,7 @@ BOOL http_response_is_websocket(HttpContext* http, HttpResponse* response)
 	if (!winpr_Digest_Init(sha1, WINPR_MD_SHA1))
 		goto out;
 
-	if (!winpr_Digest_Update(sha1, http->SecWebsocketKey, sizeof(http->SecWebsocketKey)))
+	if (!winpr_Digest_Update(sha1, http->SecWebsocketKey, strlen(http->SecWebsocketKey)))
```

Comment:
you already fixed that previously by making http->SecWebsocketKey of type BYTE, but now gcc complains about the second argument being a char*. I suggest casting that to BYTE. strlen should be correct.

Reply:
You'r right - thx for pointing out